src/time_histogram.cpp

```diff
@@ -184,8 +184,8 @@ dsec_t  CTimeHistogram::get_average_latency() {
 }
 
 
-int64_t CTimeHistogram::get_usec(dsec_t d) {
-    return (int64_t)(d*1000000.0);
+uint32_t CTimeHistogram::get_usec(dsec_t d) {
```

Comment:
this and a few following changes is a reversing of my previous patch.  since we do not admit packets with mismatched timestamps, unsigned values are OK